modules/pytket-quantinuum/tests/api1_test.py

```diff
@@ -348,31 +350,26 @@ def test_device_family(
         headers={"Content-Type": "application/json"},
     )
 
-    family_backend = QuantinuumBackend(
+    backend = QuantinuumBackend(
         device_name=chosen_device,
     )
-    family_backend._api_handler = mock_quum_api_handler
+    backend._api_handler = mock_quum_api_handler
 
     circ = Circuit(2, name="batching_test").H(0).CX(0, 1).measure_all()
-    circ = family_backend.get_compiled_circuit(circ)
-
-    kwargs = {}
-    if max_batch_cost is not None:
-        kwargs["max_batch_cost"] = max_batch_cost
-    family_backend.process_circuits(
-        circuits=[circ, circ], n_shots=10, valid_check=False, **kwargs
-    )
-
-    submitted_json = {}
-    if requests_mock.last_request:
-        submitted_json = requests_mock.last_request.json()
+    circ = backend.get_compiled_circuit(circ)
 
+    max_batch_cost = 20
     if chosen_device == "H1":
```

Comment:
Is it planed to ad a support for this to H1 or will that not happen?

Reply:
this is just to check that check for batching capability is working, if support is added by the API it will automatically be enabled by the backend